docs/gen-1/red-blue/main-glitchless/race-route/README.md

```diff
@@ -1,21 +1,21 @@
 # Pokemon Red Glitchless Race Route (NO IT-FlexLass-3Repels-PPUp)
 
-**This is one of the many race routes people use and most of them are as good. I decided to document this because it was used by both 3 finalist at the 2020 Red Tournament**
-**You can find the VOD of the Finals[here](https://youtu.be/owoeSjzdqL0), it's a good example of this route in action**
+**This is one of the many race routes people use and most of them are as good. I decided to document this because it was used by all 3 finalists at the 2020 Red Tournament.**  
```

Comment:
extra space at the end here

Reply:
Spaces at the end of a line in markdown are used to make a new line in the same paragraph and i believe thats what was intended here